test/util/server/server.go

```diff
@@ -470,11 +475,6 @@ func startKubernetesAPIServer(masterConfig *configapi.MasterConfig, clientConfig
 		return err
 	}
 
-	serverURL, err := url.Parse(fmt.Sprintf("https://%s", masterConfig.ServingInfo.BindAddress))
```

Comment:
Keep this as well because the ordering is better this way.

Reply:
got it, done